config/.env.example

```diff
@@ -0,0 +1,41 @@
+#####################
```

Comment:
i would rename this file `./config/env.example`
i do not think it's useful to hide it when it's only an example :yum:

Reply:
Mmm actually it's quite common to name it `.env.example`, so even if what you are saying makes sense, I'll leave it like that

Reply:
okok :+1: